backend/src/v5/models/users.js

```diff
@@ -207,4 +209,35 @@ User.getAvatar = async (username) => {
 
 User.uploadAvatar = (username, avatarBuffer) => updateUser(username, { $set: { 'customData.avatar': { data: avatarBuffer } } });
 
+User.resetPasswordToken = async (username) => {
```

Comment:
Naming is a bit ambiguous here. I think what we're doing is not resetting a password token, but generating a token

```suggestion
User.generateResetPasswordToken = async (username) => {
```